addons/l10n_gcc_invoice/views/report_invoice.xml

```diff
@@ -344,27 +344,40 @@
                                             <t t-if="tax_totals['display_tax_base']">
                                                 <td class="text-end o_price_total">
-                                                    <span class="text-nowrap" t-esc="amount_by_group['formatted_tax_group_amount']"/>
+                                                    <span class="text-nowrap" t-esc="amount_by_group[1]['formatted_tax_group_amount']"/>
                                                 </td>
                                                 <td class="text-end">
-                                                    <span t-esc="amount_by_group['tax_group_name']"/>
-                                                    <span class="text-nowrap"> on
-                                                        <t t-esc="amount_by_group['formatted_tax_group_base_amount']"/>
-                                                    </span>
+                                                    <strong>
+                                                        <span t-esc="amount_by_group[1]['tax_group_name']"/>
+                                                        <span class="text-nowrap"> on
+                                                            <t t-esc="amount_by_group[1]['formatted_tax_group_base_amount']"/>
+                                                        </span>
+                                                        <span class="text-nowrap">/ 
+                                                            <t t-esc="group_name_arab"/>
+                                                        </span> 
+                                                    </strong>
                                                 </td>
                                             </t>
                                             <t t-else="">
                                                 <td class="text-end o_price_total">
-                                                    <span class="text-nowrap" t-esc="amount_by_group['formatted_tax_group_amount']" />
+                                                    <span class="text-nowrap" t-esc="amount_by_group[1]['formatted_tax_group_amount']"/>
+                                                </td>
+                                                <td class="text-end">
+                                                    <strong>
+                                                        <span class="text-nowrap" t-esc="amount_by_group[1]['tax_group_name']"/>
+                                                        <span class="text-nowrap">/ 
+                                                            <t t-esc="group_name_arab"/>
+                                                        </span> 
+                                                    </strong>
```

Comment:
Unless I'm missing something, I believe there is a repetition of code here. The section inside the `t-if="tax_totals['display_tax_base']"` is very similar to what is done in the ` <t t-else="">`. I know it was already like this but let's take the opportunity to clean this now, again only if I'm not missing something. I was thinking something like this:

```html 
<td class="text-end o_price_total">
    <span class="text-nowrap" t-esc="amount_by_group['formatted_tax_group_amount']"/>
</td>
<td class="text-end">
    <strong>
        <span t-esc="amount_by_group['tax_group_name']"/>
        <t t-if="tax_totals['display_tax_base']">
            <span class="text-nowrap"> on
                <t t-esc="amount_by_group['formatted_tax_group_base_amount']"/>
            </span>
        </t>
        <span class="text-nowrap">/
            <!-- Arabic translation of tax group -->
            <t t-esc="o_sec.tax_totals['groups_by_subtotal'][o_sec.tax_totals['subtotals'][subtotal_index]['name']][amount_by_group_index]['tax_group_name']"/>
        </span>
    </strong>
</td>
```

Reply:
It works when displaying multiple tax group names so I implemented the suggestion!